Casks/bloop.rb

```diff
@@ -0,0 +1,29 @@
+cask "bloop" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "0.3.4"
+  sha256 arm:   "5142386ff9f8b4cda6d826256da6870ac4b0a1fd0402e421520bcf671d8d4578",
+         intel: "600fa0953cd5cd93fe9941771b6df5708ed46fae3a3dac1de6e948697d7089e7"
+
+  url "https://github.com/BloopAI/bloop/releases/download/v#{version}/bloop_#{version}_#{arch}.dmg",
+      verified: "github.com/BloopAI/bloop/"
+  name "bloop"
+  desc "Fast code search engine written in Rust"
```

Comment:
```suggestion
  desc "Code search engine"
```

Based on the [`desc` stanza guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc), we should avoid using empty marketing jargon, and the reference to "Rust" is not super-relevant.

Reply:
Oh ok understandable 👍